pandas/core/indexing.py

```diff
@@ -273,10 +273,10 @@ def _convert_slice_indexer(self, key: slice, axis: int):
         ax = self.obj._get_axis(min(axis, self.ndim - 1))
         return ax._convert_slice_indexer(key, kind=self.name)
 
-    def _has_valid_setitem_indexer(self, indexer):
+    def _has_valid_setitem_indexer(self, indexer) -> bool:
         return True
 
-    def _has_valid_positional_setitem_indexer(self, indexer):
+    def _has_valid_positional_setitem_indexer(self, indexer) -> bool:
```

Comment:
it looks like this always either raises or returns True; is that accurate?  if so, maybe get rid of the return value altogether (might need to update usages)

Reply:
I don't feel confident enough in reformatting this function, I'll leave it for someone else who will probably won't break the package.